tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -55,7 +68,9 @@ class CompletionMatchSuite extends BaseCompletionSuite {
 
   // Assert that Workday/Weekend symbols from previous test don't appear in result.
   checkEdit(
-    "stale2",
+    "stale2".tag(
+      IgnoreScala3
+    ), // needs fix for scala3 (adding `isStale` filter)
```

Comment:
```suggestion
    "stale2"
```
this should work since we merged the other PR